adr/0002-@covers-annotations.md

```diff
@@ -0,0 +1,36 @@
+# `@covers` annotations usage
+
+### Context
+
+PHPUnit offers a range of `@covers` annotations with the possible to enforce a strict mode or to
+enforce them. The question is when should those annotations be enforced and/or if we need to enable
+another settings as well?
+
+
+### Decision
+
+Since we are using the [`symfony/phpunit-bridge`][phpunit-bridge], we decide to leverage the
+`Symfony\Bridge\PhpUnit\CoverageListener` in `phpunit.xml.dist` in order to avoid to require the
```

Comment:
```suggestion
[`Symfony\Bridge\PhpUnit\CoverageListener`](https://symfony.com/doc/current/components/phpunit_bridge.html#code-coverage-listener) in `phpunit.xml.dist` in order to avoid to require the
```